packages/coordinator/tests/services/translation/request/dispense/dispense-notification.spec.ts

```diff
@@ -243,6 +243,21 @@ describe("fhir MedicationDispense maps correct values in DispenseNotification wh
   })
 })
 
+describe("fhir MedicationDispense throws error for DispenseNotification", () => {
+  let dispenseNotification: fhir.Bundle
+  const testFileDir = "../../tests/resources/test-data/fhir/dispensing/"
+
+  test("missing value of ODS code for reimbursement authority", () => {
+    const testFileName = "Process-Request-Dispense-Notifications-No-Ods-Value-for-ReimbursementAuthority.json"
+    dispenseNotification = TestResources.getBundleFromTestFile(testFileDir + testFileName)
+    expect(() => {
+      convertDispenseNotification(dispenseNotification, logger)
+    }).toThrow(// eslint-disable-next-line max-len
```

Comment:
again don't think this eslint comment is necessary

Reply:
You are correct. Removed.